client/src/index.jsx

```diff
@@ -1,24 +1,51 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import axios from 'axios';
+import ReviewBlurb from './ReviewBlurb.jsx'
 
 class App extends React.Component {
   constructor(props) {
     super(props)
     this.state = {
-      test: ''
+      averages: '',
+      reviews6: [],
+      users: {},
+      totalReviews: '',
     }
   }
   componentDidMount() {
-    axios('/test')
+    axios('/propId/18')
     .then(res =>{
       console.log(res.data)
+      let usersObj = {}
+      res.data.users.forEach(user => {
```

Comment:
I'm a little confused about the strategy here with users. My understanding based on this request is that you receive an array of User objects specific to the property. It also looks like the Review model includes a userId property. I would suggest querying your database for the reviews and include user information on the review object. Something like this:
     Review.findAll({where: {propertyId: id}, include: [{model: User}]})

This way you can iterate over the reviews array and pass each review as you are already doing without having to also pass the object with every user as a prop.

Reply:
more here https://gist.github.com/zcaceres/83b554ee08726a734088d90d455bc566

Reply:
@manedurphy Thank you, this is a really good idea! I tried adding the "include" line in my Review.findAll query but got the error "User is not associated to Review," because I have not associated the two tables yet. I will look up associating tables and then try it again. Thanks!